### DISCUSSION TITLE
HEIF photos converted to JPG are over exposed

### DISCUSSION THREAD

**Discussion by Opethel** [photoprism/photoprism#3185] [Q&A]
If I download the original out of photoprism (JPG converted from HEIF) and compare to the original from an IPhone 14 (both JPG, displayed on same screen), the photoprism photos are always over exposed. Has anyone else experience this? Is there something I should change in the configuration to stop this?

**Comment by lastzero** [comment]
If you already have JPEGs, why do you let PhotoPrism create JPEGs from your HEIC files in the first place?

**Reply by Opethel** [reply]
They are all originally HEIF. I was trying to show that I was comparing Photoprism HEIF to JPG vs Iphone HEIF to JPG (when you share from Iphone it converted automatically). I do not store them as JPEGs on my phone. Hopefully that makes sense...

**Reply by lastzero** [reply]
Understood. The result when converting from one format to another will never be the same. We use heif-convert for conversion and not proprietary Apple tools, unless you're running PhotoPrism natively on a Mac. Also, thumbnail colors are converted and displayed in SRGB (not Display P3) as we currently can't embed icc color profiles in thumbnails. That's planned though.

**Reply by Opethel** [reply]
Thank you for the response. I was able to make it substantially better by fixing some monitor settings as well (adaptive contrast) which was my mistake. All-in-all still very good, thanks!
